horizontal_flattening.py

```diff
@@ -30,7 +30,7 @@ def horizontal_flat(self, document):
                 del document[field]
                 for result in list(flatenned_results.keys()):
                     document[result] = flatenned_results[result]
-            if isinstance(document[field], list) and self.flatten_arrays_get_value() > 0:
+            elif isinstance(document[field], list) and self.flatten_arrays_get_value() > 0: #bug_7
```

Comment:
In the comment you've mentioned that **elif** was replaced with **if**. In the code, however, **if** has been replaced with **elif** which seems correct w.r.t. bug you mentioned. Update the PR comment to avoid confusion. Code looks good, I'll merge it.

Reply:
Oh yes i think typo. If has been replaced by elif